LEGO1/lego/legoomni/src/race/legoracers.cpp

```diff
@@ -451,14 +476,45 @@ MxResult LegoRaceCar::VTable0x94(LegoPathActor* p_actor, MxBool p_bool)
 	return SUCCESS;
 }
 
-// STUB: LEGO1 0x10013600
+// FUNCTION: LEGO1 0x10013600
+// FUNCTION: BETA10 0x100cbe60
 MxResult LegoRaceCar::VTable0x9c()
 {
-	// TODO
-	return SUCCESS;
+	MxResult result;
+
+	if (m_userNavFlag) {
+		result = LegoCarRaceActor::VTable0x9c();
+
+		if (m_boundary) {
+			MxS32 bVar2 = 0;
```

Comment:
The compiler entropy/register allocation is fixed when moving this variable declaration outside of the `if` block, but the diff looks less correct.